packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -193,9 +195,6 @@
       <DeploymentContent>false</DeploymentContent>
     </Text>
   </ItemGroup>
-  <ItemGroup>
-    <Natvis Include="$(ReactNativeWindowsDir)\Folly\Folly.natvis" />
```

Comment:
why are we removing the natvis ?

Reply:
App shouldn't need to reference this since they're consuming Microsoft.ReactNative, not Folly directly. Right?

Reply:
IF so, this should be added to the shared targets for everyone to enjoy, not just this one app.

Reply:
I believe this file is used by VS to format Folly objects in the debugger, it's not used for building the app or at runtime

Reply:
Users of MSRN shouldn't directly touch Folly, so I can understand this being removed, but the natvis is useful for engineers on RNW debugging internals. Don't think it's needed in something user-facing though.